protocol/indexer/protocol/v1/v1_mappers.go

```diff
@@ -176,6 +176,8 @@ func ConvertToClobPairStatus(status clobtypes.ClobPair_Status) ClobPairStatus {
 		return ClobPairStatus_CLOB_PAIR_STATUS_POST_ONLY
 	case clobtypes.ClobPair_STATUS_INITIALIZING:
 		return ClobPairStatus_CLOB_PAIR_STATUS_INITIALIZING
+	case clobtypes.ClobPair_STATUS_FINAL_SETTLEMENT:
+		return ClobPairStatus_CLOB_PAIR_STATUS_FINAL_SETTLEMENT
 	default:
 		panic("invalid clob pair status")
```

Comment:
Not an issue with your code, but mind including function name + CLOB pair status in the panic?